Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG PYTHON_VERSION=3.8
+FROM python:${PYTHON_VERSION} AS base
+
+WORKDIR /app
+
+COPY ./accounts ./accounts
+COPY ./api ./api
+COPY ./lists ./lists
+COPY ./todolist ./todolist
+COPY manage.py requirements.txt ./
+
+FROM python:${PYTHON_VERSION}-slim
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1
+
+COPY --from=base /app ./
+
+RUN pip install --upgrade pip && \
+    pip install -r requirements.txt
+
+RUN python manage.py migrate
+
+EXPOSE 8080
+
+ENTRYPOINT ["python", "manage.py", "runserver", "0.0.0.0:8080"]
```

Comment:
![image](https://github.com/user-attachments/assets/4bc666ab-85bf-4225-b96c-afd11cb3cb5f)
Add Eof(End-of-file), need to add a blank line at the end